src/stores/battlefieldStore.ts

```diff
@@ -0,0 +1,166 @@
+import { action, computed, observable } from 'mobx';
+import { Tank } from '../classes/tank';
+import { Point } from '../zone/point';
+import { ZoneShape } from '../zone/zoneShape';
+import { Bullet } from './../classes/bullet';
+import { BulletStore } from './bulletStore';
+import { TankStore } from './tankStore';
+
+
+const MIN_SCALE = 20;
+const MAX_SCALE = 40;
+
+export let SCALE_COEF = observable.box(MIN_SCALE);
+export let TRANSITION = observable.box(1);
```

Comment:
keep in mind that your constants shouldn't be changeable (in other case it is not a constant)